app/main/routes.py

```diff
@@ -84,14 +84,64 @@ def callback():
     session["token_scope"] = access_token_response["scope"]
     session["session_state"] = access_token_response["session_state"]
 
-    return redirect(url_for("main.poc_search"))
+    return redirect(url_for("main.browse"))
 
 
 @bp.route("/accessibility", methods=["GET"])
 def accessibility():
     return render_template("accessibility.html")
 
 
+@bp.route("/browse", methods=["POST", "GET"])
+@access_token_sign_in_required
+def browse():
+    transferring_body_id = request.args.get("transferring_body", None)
+    series_id = request.args.get("series", None)
+    consignment_id = request.args.get("consignment_id", None)
+    query = request.form.get("query", None)
+
+    form = SearchForm()
+    search_results = []
+    search_type = "browse"
+
+    # Create a list to accumulate filter conditions
+    filters = []
+
+    if transferring_body_id:
+        filters.append({"transferring_body_id": transferring_body_id})
+        search_type = "transferring_body"
+    if series_id:
+        filters.append({"series_id": series_id})
+        search_type = "series"
+    if consignment_id:
+        filters.append({"consignment_id": consignment_id})
+        search_type = "consignment"
+    if query:
+        filters.append({"query": query})
+        search_type = "browse"
+
+    # Apply filters to browse_data or call fuzzy_search based on search_type
+    if search_type == "browse" and query:
```

Comment:
like we discussed with Terry, doing a search on the browse view should lead to a search results page which we havent yet created but should be something like the poc-search view. 
Maybe we can just point to that for now in this browse template, that way we can remove the need for confusing this view with search logic?

Reply:
Good shout, removed integrating the search logic into browse for now.